assets/src/components/v2/triptych/README.md

```diff
@@ -0,0 +1,82 @@
+# Triptych app packaging
+
+- Ensure [Corsica](https://hexdocs.pm/corsica/Corsica.html) is used on the server to allow CORS requests (ideally limited to just the triptych-relevant routes). It should already be configured at [this line](/lib/screens_web/controllers/v2/screen_api_controller.ex#L9) in the API controller--if it is, you don't need to do anything for this step.
+- Double check that any behavior specific to the triptych screen environment happens inside of an `isOFM()` check. This includes:
+  - `buildApiPath` in use_api_response.tsx should return a full URL for the API path: prefix `apiPath` string with "https://screens.mbta.com".
+  - `imagePath` in util.tsx should return relative paths (no leading `/`).
+- Create priv/static/triptych-app.html if it doesn’t already exist. Copy paste the following contents in:
+
+  ```html
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Screens</title>
+      <link rel="stylesheet" href="triptych_v2.css" />
+    </head>
+
+    <body>
+      <div
+        id="app"
+        data-last-refresh="2020-09-25T17:23:00Z"
+        data-environment-name="screens-prod"
+      ></div>
+      <script type="text/javascript" src="polyfills.js"></script>
+      <script type="text/javascript" src="triptych_v2.js"></script>
+    </body>
+  </html>
+  ```
+
+- Set the version string in assets/src/components/v2/triptych/version.tsx to `current_year.current_month.current_day.1`.
+- In assets/webpack.config.js, change `publicPath` in the font config to have value `'fonts/'`.
+- **Only if you are packaging for local testing**
+  - replace `const playerName = useOutfrontPlayerName();` in assets/src/apps/v2/triptych.tsx with `const playerName = "BKB-TRI-001";` (or any other player name from one of the triptych screen IDs (`TRI-${playerName}`)). This data is provided by Outfront's "wrapper" app that runs on the real triptych screens, but we need to set it ourselves during testing. Think of it as a sort of frontend environment variable.
```

Comment:
That example player name is probably not valid—I just made a guess based on what some of the DUP player names look like. We'll need to update this line once we have a real player name to work with.